Dockerfile.frontend-container

```diff
@@ -1,19 +1,18 @@
 FROM node:16-alpine
 
 WORKDIR /usr/src/app
-
-COPY tornjak-frontend .
-
-RUN npm install --omit=dev
-
-RUN npm prune --production
+CMD ["npm", "start"]
 
 # set env variables
-ENV REACT_APP_API_SERVER_URI $REACT_APP_API_SERVER_URI
-ENV REACT_APP_AUTH_SERVER_URI $REACT_APP_AUTH_SERVER_URI
-ENV REACT_APP_SPIRE_HEALTH_CHECK_ENABLE $REACT_APP_SPIRE_HEALTH_CHECK_ENABLE
+ENV REACT_APP_API_SERVER_URI $REACT_APP_API_SERVER_URI \
+    REACT_APP_AUTH_SERVER_URI $REACT_APP_AUTH_SERVER_URI \
+    REACT_APP_SPIRE_HEALTH_CHECK_ENABLE $REACT_APP_SPIRE_HEALTH_CHECK_ENABLE \
+    PORT_FE=3000
```

Comment:
changes from 4 independent layers to just a single layer.

Reply:
Thank you for this layer improvement!